lib/interactive.dart

```diff
@@ -155,6 +155,8 @@ class Interactive {
       ..setAttribute("y2", "${CHART_HEIGHT - CHART_XAXIS_HEIGHT}");
     svgContainer.append(xAxisLine);
 
+    var groupPeople = Map<String, List<model.Person>>();
```

Comment:
groupPeople doesn't really make sense to me.

peopleByLabel?

Reply:
Yes peopleByLabel is much better - will take note on the variable names. Fixed